Source/IntegrationTests/TestFiles/LitTests/LitTest/at-attributes/at-attributes-typos.dfy.expect

```diff
@@ -0,0 +1,15 @@
+at-attributes-typos.dfy(16,9): Error: the parameter named 'low' is already given positionally
+at-attributes-typos.dfy(14,9): Error: Argument to attribute Compile must be a literal
+at-attributes-typos.dfy(13,20): Error: wrong number of arguments (got 2, but attribute 'Compile' expects at most 1: (0: bool))
+at-attributes-typos.dfy(12,15): Error: incorrect argument type for attribute parameter '0' (expected bool, found string)
+at-attributes-typos.dfy(28,0): Error: @Fuel attribute cannot be applied to method
+at-attributes-typos.dfy(42,31): Error: wrong number of arguments (got 1, but attribute 'IsolateAssertions' expects 0)
+at-attributes-typos.dfy(22,0): Error: @Fuel attribute cannot be applied to datatype
```

Comment:
Great, thanks for adding the filtering. This is good enough for now, but in the next iteration let's be more precise and helpful: "@Fuel can only be applied to functions, but Useless is a datatype"

(that will require modeling the filter as a subset of attribute-able program elements of course, instead of a free-form predicate)